_posts/2021-03-03-cockpit-239.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 239
+author: lis
+date: 2021-03-03
+tags: cockpit linux
+slug: cockpit-239
+category: release
+summary: Cockpit with restyled updates page
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 239.
+
+### Machines: Allow editing disk cache mode
+
+It's now possible to edit the cache mode for existing virtual machine
+images.  This was previously only possible when creating new images.  In
+order to edit the cache mode, any machines using the volume need to be
+powered off.
+
+### Packaging: Removed ./configure options for distribution specific packages
+
+The previous --with-vdo-package, --with-nfs-client-package,
+--with-appstream-config-packages, and --with-appstream-data-packages
+options for ./configure got removed. This information is now contained
+in the respective manifest files, for example for the Storage page. This
+makes the manifest files portable across distributions. They can still
+be adjusted by administrators through override.json files.
+
+### Logs: add link to the services page
+
+When viewing the details of a log entry, there's now a link directly to
+the relevant service page, if applicable.
+
+### SELinux: Restyle to fit with other pages
+
+The SELinux page has been restyled to better match the style of other
+pages in Cockpit.
```

Comment:
A screenshot could not hurt (not that important, of course). But the one in https://github.com/cockpit-project/cockpit/pull/15364 is too large and wide.

Reply:
Oh cool! I somehow missed this. Looks great!

Reply:
I've made my own screenshot for this one